lib/sqeduler/trigger_lock.rb

```diff
@@ -15,6 +15,8 @@ def lock
       # - return true if already acquired
       # - refresh the lock if already acquired
       refresh || super
+    rescue RuntimeError
```

Comment:
Why `RuntimeError`? I would probably just do `rescue` in order to catch `StandardError` and descendants.